.github/ci.yml

```diff
@@ -1,5 +1,4 @@
 name: CI
```

Comment:
I'm not understanding it. I think you should upgrade it, not downgrade it. Also, it should be in the workflow folder.